packages/core/src/model.ts

```diff
@@ -389,6 +389,13 @@ export class ChartModel {
 		return this.colorScale;
 	}
 
+	/**
+	 * For charts that might hold an associated status for their dataset
+	 */
+	getStatus() {
+		return null;
+	}
```

Comment:
do we really need this for the base model class?

Reply:
yeah so that i can extend it in model-meter?